Glossary (#109)
Agoric/documentation: main/glossary/README.md

> # ERTP Glossary

Reviewer: I agree with changing the title

> # ERTP Glossary

Author: addressed in latest commit

> This page lists words, expressions or concepts that are used by the Agoric technology stack

Reviewer: Also an Oxford comma after "expressions" (yeah, I'm one of those people). I'd also tighten it by removing "that are".

> This page lists words, expressions or concepts that are used by the Agoric technology stack

Author: addressed in latest commit

> Also an Oxford comma after "expressions" (yeah, I'm one of those people)

My native language is French; the Oxford comma debate feels very foreign to me :-p
This makes no difference to me, so i trust you and made the change :-)

Extent must be Comparable.

## Handle

> A handle is a unique identifier implemented as an JavaScript object. Only its identity is meaningful, so most handles do not properties. Unlike number or string identifiers, handles are unforgeable. This means no one who doesn't already have reference to the handle can create a fake a new reference to it.

Reviewer 2: ```suggestion
A handle is a unique identifier implemented as a JavaScript object. Only its identity is meaningful, so most handles do not have properties. Unlike number or string identifiers, handles are unforgeable. This means no one who doesn't already have reference to the handle can create a fake a new reference to it.
```

> A handle is a unique identifier implemented as a JavaScript object. Only its identity is meaningful, so most handles do not have properties. Unlike number or string identifiers, handles are unforgeable. This means no one who doesn't already have reference to the handle can create a fake a new reference to it.

Reviewer 3: No handles have properties.
```suggestion
A handle is a unique identifier implemented as a JavaScript object. Only its identity is meaningful, so handles do not have properties. Unlike number or string identifiers, handles are unforgeable. This means the only way to know a handle identity is being given an object reference, and no identity can be guessed and no fake identity will succeed. 
```

> For example, Zoe contracts have an `offerHandle`. The `offerHandle` is the key for requesting the current allocation for this offer or reallocating the offerHandle's assets.

Reviewer 3: ```suggestion
For example, Zoe often uses `offerHandle` to refer to offers. Zoe contracts can use an offer's `offerHandle` as the key for requesting the current allocation of an offer or reallocating the offer's assets.
```

> For example, there exists an `offerHandle` is Zoe contracts. The `offerHandle` is the key to knowing what the current allocation for this offer is or to reallocate assets to this offerHandle.

Reviewer 2: ```suggestion
For example, Zoe contracts have an `offerHandle`. The `offerHandle` is the key for requesting the current allocation for this offer or reallocating the offerHandle's assets.
```

## Issuer

> Can create empty purses and payments, but it cannot mint new amounts. The issuer can also transform payments (splitting payments, combining payments, burning payments, and claiming payments exclusively). The issuer should be gotten from a trusted source and then relied upon as the decider of whether an untrusted payment is valid.

Reviewer: ## Issuer

Can create empty purses and payments, but cannot mint new amounts. Issuers can also transform payments (splitting, combining, burning, and claiming payments exclusively). You should get an issuer from a trusted source and then rely on it to determine if an untrusted payment is valid. Issuers are linked to a single mint and vice versa, so each issuer only works with one type of thing, such as only working with quatloos or only working with moola.

Reviewer: Please check with someone to make sure that last is accurate. If it is, I think it's an important aspect of an issuer that should be mentioned in its definition.

Reviewer 3: Your changes looks correct to me @tyg.

Author: I copied/pasted
